data/json/monsters/reptile_amphibian.json

```diff
@@ -471,5 +404,270 @@
+    "name": { "str": "foul toad" },
+    "reproduction": { "baby_egg": "egg_foul_toad", "baby_count": 8, "baby_timer": 36 },
+    "//": "When possible, add effect for aoe morale debuff around toad. Possibly stack up until triggering nausea/vomitting."
+  },
+  {
+    "id": "mon_freedom_toad",
+    "type": "MONSTER",
+    "copy-from": "mon_proxy_frog",
+    "description": "American toads can vary greatly in appearance.  Individual toads will shift their coloration and patterning throughout the seasons to better camouflage.",
+    "name": { "str": "american toad" },
+    "reproduction": { "baby_egg": "egg_freedom_toad", "baby_count": 8, "baby_timer": 36 },
+    "baby_flags": [ "SPRING", "SUMMER" ]
+  },
+  {
+    "id": "mon_odd_toad",
+    "type": "MONSTER",
+    "copy-from": "mon_freedom_toad",
+    "description": "There's something very off about this toad's proportions.  No one feature stands out under scrutiny but it looks like it hasn't fully grown.",
+    "name": { "str": "odd toad" },
+    "upgrades": { "age_grow": 120, "into": "mon_toad_small" },
+    "reproduction": { "baby_egg": "egg_odd_toad", "baby_count": 8, "baby_timer": 36 }
+  },
+  {
+    "id": "mon_peeper_frog",
+    "type": "MONSTER",
+    "copy-from": "mon_proxy_frog",
+    "description": "This diminutive frog has a distinctive chirping call and bands of darker skin that form a rough X shape on its back.",
+    "name": { "str": "spring peeper" },
+    "volume": "20 ml",
+    "weight": "4 g",
+    "hp": 1,
+    "reproduction": { "baby_egg": "egg_peeper_frog", "baby_count": 8, "baby_timer": 20 },
+    "baby_flags": [ "SPRING", "SUMMER" ]
+  },
+  {
+    "id": "mon_vocal_frog",
+    "type": "MONSTER",
+    "copy-from": "mon_peeper_frog",
+    "description": "Like a powered machine, this frog never stops making noise.  When it calls, its whole torso expands like an inflatable suit.",
+    "name": { "str": "vocal treefrog" },
+    "reproduction": { "baby_egg": "egg_vocal_frog", "baby_count": 8, "baby_timer": 20 },
+    "//": "When possible, add frequent parroting."
+  },
+  {
+    "id": "mon_gray_frog",
+    "type": "MONSTER",
+    "copy-from": "mon_proxy_frog",
+    "description": "This treefrog can change its coloration much like a chameleon, shifting from a near-white gray, through greens and browns, to a dark, almost charcoal color.",
+    "name": { "str": "gray treefrog" },
+    "volume": "150 ml",
+    "weight": "10 g",
+    "hp": 2,
+    "reproduction": { "baby_egg": "egg_gray_frog", "baby_count": 8, "baby_timer": 26 },
+    "baby_flags": [ "SPRING", "SUMMER" ],
+    "extend": { "flags": [ "CAMOUFLAGE" ] }
+  },
+  {
+    "id": "mon_shift_frog",
+    "type": "MONSTER",
+    "copy-from": "mon_gray_frog",
+    "description": "You've seen it wrinkle its skin into bark and disappear on a tree; settle on a toadstool and take its spots.  It's no trick of the light, this frog's skin shifts not just color but texture.",
```

Comment:
Is this supposed to be like how octopus camouflage works? I initially typed this thinking you went too deep into the nether territory but if it is supposed to be octopus-esque I think it's fine as a mutation. The only issue I have with this and all the other weird frogs is they produce non-mutant meat. They likely should not, even if they're just mild mutants.